parity/cli/usage.txt

```diff
@@ -294,6 +296,10 @@ Sealing/Mining Options:
                                    means we relay nothing if not mining);
                                    lenient - Same as strict when mining, and cheap
                                    when not (default: {flag_relay_set}).
+  --min-gas-price WEI              Minimum amount of Wei per GAS to be paid for a
```

Comment:
since this is exactly equivalent to `--gasprice`, `--gasprice`'s definition should be changed to `Equivalent to --min-gas-price WEI`.